### DISCUSSION TITLE
Climate Entity - multiple heat settings - multi-stage AUX_HEAT?

### DISCUSSION THREAD

**Discussion by jeeftor** [home-assistant/architecture#791] [Entity Models]
I have a gas fireplace and I've written an integration [IntelliFire](https://www.home-assistant.io/integrations/intellifire/). 

Depending where you live in the world you might use just a gas fireplace as a heating element for your home. The newer ones support a thermostatic mode in which they will tweak the flame level depending on how far from the thermostat target the device is. 

Right now climate entities appear to support two stage heating (likely to support heat pumps):

* `HVACMode.HEAT` (is the heat on - stage 1)
* `SUPPORT_AUX_HEAT` (is stage 2 heat on)

I wanted to open a discussion as to whether there was any reason to change the `AUX_HEAT` to allow multiple stages? My individual unit has 5 levels of flame. I've struggled for the best way to support this with my device. I've created buttons to support multiple flame heights - but I was wondering if there is any reason that the CLIMATE entity should support an `AUX_HEAT_LEVEL`

Given the world is transitioning off of Gas in some regions I don't know if this is a worthy endeavor - however - for my specific use case it may be very useful.

**Comment by gjohansson-ST** [comment]
It sounds more to me you should implement a `select` entity to control the flame.
I think the use-case for what you propose would be very limited.

**Comment by frenck** [comment]
Related to this, I've opened up a proposal to remove auxiliary heating from climate altogether: <https://github.com/home-assistant/architecture/discussions/932>
